src/main/java/uk/gov/hmcts/darts/common/entity/UserAccountEntity.java

```diff
@@ -82,6 +78,30 @@ public class UserAccountEntity extends CreatedModifiedBaseEntity {
         inverseJoinColumns = {@JoinColumn(name = "grp_id")})
     private Set<SecurityGroupEntity> securityGroupEntities = new LinkedHashSet<>();
 
+    @NotAudited
+    @Column(name = "user_os_name")
+    private String userOsName;
+
+    @NotAudited
+    @Column(name = "user_ldap_dn")
+    private String userLdapDomainName;
+
+    @NotAudited
+    @Column(name = "user_global_unique_id")
+    private String userGlobalUniqueId;
+
+    @NotAudited
+    @Column(name = "user_login_name")
+    private String userLoginName;
+
+    @NotAudited
+    @Column(name = "user_login_domain")
+    private String userLoginDomain;
+
+    @NotAudited
+    @Column(name = "user_state")
+    private Short userState;
```

Comment:
What is the validation on this fields/

NotNull Max Lenght, Min Length, Regex checks etc?

Reply:
I think it makes it easier to identify what validation is applied to the entity so that when you are scripting you can really easily / quickly identify what a valid value should look like, without having to go to the data model, and it helps with belt and braces as we enforce the restriction across the board

I do agree with the inconsistencies that can bring so I will leave this up to your discretion :)

Reply:
On the basis the datamodel is now mature and further changes should (!) be minimal, I have now included the data layer constraints on the Entities. These are limited to `nullable = false`, and `length = n` - we don't know the full shape of the data so I'd rather not overconstrain with min length or regex checks.